src/pages/dar_application/ResearcherInfo_new.js

```diff
@@ -0,0 +1,497 @@
+          Alert({
+            id: 'profileSubmitted',
+            type: 'important',
+            title: span([
+              `You must submit `, profileLink, ` and obtain a `, libraryCardLink,
+              ` from your Signing official before you can submit a Data Access Request.`
+            ])
+          })
+        ]),
+
+        h2('Step 1: Researcher Information'),
+
+        div({className: 'dar-application-row'}, [
+          h(FormField, {
+            id: `researcher_name`,
+            placeholder: 'Enter Firstname Lastname',
+            title: '1.1 Researcher',
+            validators: [FormValidators.REQUIRED],
+            ariaLevel: ariaLevel + 1,
+            onChange: ({key: name, value}) => formFieldChange({name, value}),
+            defaultValue: researcher,
+          }),
+        ]),
+
+        div({className: 'dar-application-row'}, [
+          h3('1.2 Researcher Identification'),
+          span({ className: 'default-color' }, [
+            'Please authenticate with ',
+            a({ target: '_blank', href: 'https://era.nih.gov/reg-accounts/register-commons.htm' }, ['eRA Commons']), ' in order to proceed.'
+          ]),
+          div({ className: 'flex-row', style: { justifyContent: 'flex-start', alignItems: 'flex-start' } }, [
+            h4({ style: { marginRight: 30, marginTop: 30 } }, '1.2.1'),
+            eRACommons({
+              destination: eRACommonsDestination,
+              onNihStatusUpdate: onNihStatusUpdate,
+              location: location,
+              validationError: showValidationMessages,
+              header: true
+            })
+          ]),
+          div({ className: 'flex-row', style: { justifyContent: 'flex-start' } }, [
+            h4({ style: { marginRight: 30 } }, '1.2.2'),
+            h(FormField, {
+              id: 'checkNihDataOnly',
+              toggleText: span({ style: { fontSize: 14, fontWeight: 'bold' }}, ['I am exclusively applying for NIH data (ex. GTex)']),
+              type: FormFieldTypes.CHECKBOX,
+              ariaLevel: ariaLevel + 2,
+              onChange: ({key: name, value}) => formFieldChange({name, value}),
+              defaultValue: checkNihDataOnly
+            })
+          ]),
+          div({ className: 'flex-row', style: { justifyContent: 'flex-start' } }, [
+            h4({ style: { marginRight: 30 } }, '1.2.3'),
+            h(FormField, {
+              id: 'checkCollaborator',
+              toggleText: span({ style: { fontSize: 14, fontWeight: 'bold' }}, ['I am an NIH intramural researcher (NIH email required)']),
+              type: FormFieldTypes.CHECKBOX,
+              ariaLevel: ariaLevel + 2,
+              onChange: ({key: name, value}) => formFieldChange({name, value}),
+              defaultValue: checkCollaborator
+            })
+          ]),
+        ]),
+
+        div({className: 'dar-application-row'}, [
+          h(FormField, {
+            id: `principal_investigator`,
```

Comment:
We have `piName` as a value in the current schema: https://github.com/DataBiosphere/consent/blob/develop/src/main/resources/data-submitter-schema_v1.json#L62 

Let's track any modifications necessary to the schema so that we can tackle them as a unit. I lean towards naming that makes the most sense from a UI dev perspective.